scalding-repl/src/main/scala/com/twitter/scalding/ReplImplicits.scala

```diff
@@ -178,7 +183,8 @@ object ReplImplicits extends FieldConversions {
    * @param source to convert to a Pipe.
    * @return a Pipe that is the result of reading the specified Source.
    */
-  implicit def sourceToPipe(source: Source): Pipe = source.read(flowDef, mode)
+  implicit def sourceToPipe(source: Source)(implicit flowDef: FlowDef, mode: Mode): Pipe =
```

Comment:
I really don't think we need this. I think it was just cargo culted from Dsl by the (wibidata folks that started the REPL).